diff --git a/node_modules/vike/dist/esm/node/runtime/renderPage/getPageAssets/retrieveAssetsDev.js b/node_modules/vike/dist/esm/node/runtime/renderPage/getPageAssets/retrieveAssetsDev.js
index 0639ba9..3b60115 100644
--- a/node_modules/vike/dist/esm/node/runtime/renderPage/getPageAssets/retrieveAssetsDev.js
+++ b/node_modules/vike/dist/esm/node/runtime/renderPage/getPageAssets/retrieveAssetsDev.js
@@ -58,14 +58,11 @@ function collectCss(mod, styleUrls, visitedModules, importer) {
         collectCss(dep, styleUrls, visitedModules, mod);
     });
 }
+// https://github.com/vikejs/vike/issues/2039
 function isStyle(mod) {
-    if (styleFileRE.test(mod.url) || (mod.id && /\?vue&type=style/.test(mod.id))) {
-        // `mod.type` seems broken
-        assert(mod.type === 'js');
-        // logModule(mod)
-        return true;
-    }
-    return false;
+    if (mod.type === 'css') return true
+    if (styleFileRE.test(mod.url) || (mod.id && /\?vue&type=style/.test(mod.id))) return true
+    return false
 }
 /*
 function logModule(mod: ModuleNode) {
